src/routes/tools/local-climate-change-snapshot/index.svelte

```diff
@@ -50,14 +50,10 @@
   />
 </svelte:head>
 
-<Header
-  iconPaths="{tool.icons}"
-  title="{tool.title}"
-  description="{toolIntro}"
```

Comment:
TODO: clean up tool intro text / description in content dir. Should probably export JS instead of having it be markdown.